pkg/build/gobuild.go

```diff
@@ -689,9 +718,32 @@ func (g *gobuild) buildOne(ctx context.Context, refStr string, base v1.Image, pl
 
 	empty := v1.Time{}
 	if g.creationTime != empty {
-		return mutate.CreatedAt(image, g.creationTime)
+		image, err = mutate.CreatedAt(image, g.creationTime)
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	si := signed.Image(image)
+
+	if !g.disableSBOM {
```

Comment:
I'd flip `disableSBOM` around to avoid the double negative here.

Reply:
I changed the flow to avoid the double negative